go.mod

```diff
@@ -5,18 +5,20 @@ go 1.20
 require (
 	github.com/hashicorp/go-version v1.7.0
 	github.com/liip/sheriff v0.12.0
+	github.com/miekg/dns v1.1.62
 	github.com/pkg/errors v0.9.1
 	github.com/schollz/progressbar/v3 v3.15.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
-	github.com/zmap/dns v1.1.59-zdns-4
 	github.com/zmap/go-iptree v0.0.0-20210731043055-d4e632617837
 	github.com/zmap/zcrypto v0.0.0-20240803002437-3a861682ac77
 	github.com/zmap/zflags v1.4.0-beta.1.0.20200204220219-9d95409821b6
 	github.com/zmap/zgrab2 v0.1.8
 	gotest.tools/v3 v3.5.1
 )
 
+replace github.com/miekg/dns => github.com/zmap/dns v1.1.63-zdns1
```

Comment:
`zmap/dns v1.1.63*` is `miekg/dns v1.1.62`. New tags are just to bypass go proxy cache system.